runtime/src/main/scala/akka/grpc/javadsl/ServerReflection.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.javadsl
+
+import java.util.Collection
+import java.util.concurrent.CompletionStage
+
+import akka.actor.ActorSystem
+import akka.annotation.ApiMayChange
+import akka.grpc.ServiceObject
+import akka.grpc.internal.ServerReflectionImpl
+import akka.http.javadsl.model.{ HttpRequest, HttpResponse }
+import akka.stream.Materializer
+
+import grpc.reflection.v1alpha.reflection.ServerReflectionHandler
+
+@ApiMayChange
+class ServerReflection(objects: Collection[ServiceObject]) {
```

Comment:
Since this doesn't really have a lifecycle and just has one method operating on the set of objects, does it really make sense to make it a class vs just a method `create(objects,  system, materializer)`?